http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20230418_update_custom_env_length.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="yonghaoy" id="update_custom_env_length">
+        <modifyDataType
+                columnName="customClusterEnvironmentVariables"
+                newDataType="text"
```

Comment:
Instead of changing to `test`, could you keep varchar, but increase to the desired size?
e.g. `newDataType="varchar(4064)"`

Reply:
I throught using text will give us a lot more space and should be good enough moving forward.
If keep using varchar but increase to 4064, I am concerned we will hit this error again as we will kept adding env vars into runtime to help user manage their dataset asset. 
Not DB expert, what is the drawbacks of changing from `varchar` to `text`?

Reply:
varchar is stored inline and usually faster to access than text

Reply:
You can increase varchar to the size you need, I was just giving you an example

Reply:
I see. Doing some research. Is [this post](https://stackoverflow.com/questions/1476987/how-to-store-more-than-255-char-in-mysql-database) correct? 
Leo is using mysql 5.0.3 or higher, and I can just set to `varchar(65535)` ?

Reply:
We are using 5.6 so this should work yes, the only other reason why I am picky is that other stakeholders are using this column as well, I do not see why changing it to text would make this noticeably slower for others but I would like to make the smallest possible change here

Reply:
SG! Thanks